lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-flavor.rb

```diff
@@ -1,4 +1,5 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Flavor < MiqAeServiceFlavor
+    expose :name
```

Comment:
@aufi 
If you compare with this https://github.com/ManageIQ/manageiq/blob/master/lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-amazon-cloud_manager-flavor.rb#L2
I don't think you have to expose the name it happens automatically